### DISCUSSION TITLE
Hello, I hope this is the right time.

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#102] [Ideas]
Hello @GoodestEnglish !
I really would like to see WindSpigot support as soon as possible.

I know that other spigot supports are planned but hear me out.
- PaperMc doesnt have kb and kb profiling.
- CarbonSpigot is expensive.
- FoxSpigot is affordable but has messed up events (we discussed in issues)
- ImanitySpigot is also not very affordable.
All these supported spigots aren't perfect.

But WindSpigot is literally the best free spigot with
*) KB modification And KB Profiling
*) Better Performance
*) Free Of Cost
*) Open source (Probably easier to add kb profile support for this)

So, I really want to see WindSpigot support as soon as possible.
Adding support for this Spigot would guarantee a popularity boost for Eden.
I hope this doesn't interfere with a lot of work you already have.

**Comment by GoodestEnglish** [comment]
Looks good, will add once I am free

**Comment by GoodestEnglish** [comment]
Added

**Comment by HierarchyMan** [comment]
Just curious is it still possible for you to add rave spigot support?

**Reply by GoodestEnglish** [reply]
I have no idea what is rave spigot. Send me the github link and I will have a look into it.

**Reply by HierarchyMan** [reply]
Here's the source: https://www.mediafire.com/file/8tmjw1045727qzl/RaveSpigot-master.rar/file

It was made by refine development it was open sourced cuz it was dethroned by carbon. 

Edit:- The source is on their discord server but it's too hard to find
![Screenshot_2023-04-14-22-22-34-86_572064f74bd5f9fa804b05334aa4f912.jpg](https://user-images.githubusercontent.com/97939376/232107716-1bbff0dc-8ce3-439f-ad57-2bf616971c6c.jpg)

Edit Edit:- I searched a lot but couldn't find an API for this spigot lemme know if it's actually needed.

**Reply by HierarchyMan** [reply]
@GoodestEnglish

**Reply by GoodestEnglish** [reply]
Sorry I missed this. Added in todo https://github.com/diamond-rip/Eden/discussions/411
